.github/workflows/ci.yaml

```diff
@@ -37,3 +40,34 @@ jobs:
         with:
           name: quarto-vscode-${{ github.sha }}
           path: ${{ steps.package_extension.outputs.vsixPath }}
+  
+
+  publish-extension:
+    runs-on: ubuntu-latest
+    if: |
+      github.event_name == 'release' &&
+      github.repository_owner == 'quarto-dev'
+    needs:
+      - package-extension
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: latest
+      - run: yarn install --immutable --immutable-cache --check-cache
+      - uses: actions/download-artifact@v4
+        with:
+          name: quarto-vscode-${{ github.sha }}
+
+      - name: Publish to Open VSX Registry
+        uses: HaaLeo/publish-vscode-extension@v1
+        with:
+          pat: ${{ secrets.OPEN_VSX_TOKEN }}
+          extensionFile: ${{ needs.package_extension.outputs.vsixPath }}
+
+      - name: Publish to Visual Studio Marketplace
+        uses: HaaLeo/publish-vscode-extension@v1
+        with:
+          pat: ${{ secrets.VS_MARKETPLACE_TOKEN }}
```

Comment:
This is the second of two tokens we will need added to the repo secrets to make this work.

Reply:
I just added this to this repo's secrets, after being added as a publisher for the Quarto extension. I made a token that expires after 90 days FYI so we will need to replace it at that time. If we feel comfortable, next time I can make one that never expires.